start.sh

```diff
@@ -3,11 +3,13 @@
 # Note: set user using `--env uid=XXXX --env gid=XXXX`, instead of using
 # docker's `--user` flag
 
-if [ ! -s "${HOME}/.conan/registry.txt" ] ; then
-    cat "${HOME}/.conan/registry.template.txt" >> "${HOME}/.conan/registry.txt"
+if [[ ! -s "${HOME}/.conan/registry.json" ]] ; then
+    conan remote add ci "https://artifactory.redlion.net/artifactory/api/conan/conan-local"
```

Comment:
This makes the whole toolchain much less generic. If we have to `-v`-mount credentials, can we mount in remotes as well?

Reply:
Mounting the registry file is encouraged, as shown in the `docker-native` script. But, if someone _doesn't_ mount in the registry file, I want it to still work.